jsk_robot_common/jsk_robot_startup/lifelog/common_logger.launch

```diff
@@ -184,8 +205,12 @@
           respawn="$(arg respawn)"
           if="$(arg save_tf)">
       <rosparam subst_value="true">
+        database: $(arg database)
         update_rate: $(arg log_rate)
       </rosparam>
+      <rosparam subst_value="true" if="$(eval arg('collection') != '')">
```

Comment:
@tkmtnt7000 can we avoid using `eval` ??
which is available from kinetic http://wiki.ros.org/roslaunch/XML

Reply:
I understand, (but it seems that `eval` makes codes easier to read...)

One solution is that we set rosparam `collection` manually  in each demo launch file instead of `arg`, like the following.

```xml
<include file="$(find jsk_robot_startup)/lifelog/common_logger.launch">
  <arg name="hoge" value="hoge" />
  ...
  <param name="collection" value="go_to_kitchen" />
</include>
```

Reply:
Thank you very much for suggestion.
I updated in [559f602](https://github.com/jsk-ros-pkg/jsk_robot/pull/1720/commits/559f602efcc67fcefbba4ff07f1982501aefdbda),[fda2d1a](https://github.com/jsk-ros-pkg/jsk_robot/pull/1720/commits/fda2d1af52e18afa2de4d573c7a86961443dfac2), [648157c](https://github.com/jsk-ros-pkg/jsk_robot/pull/1720/commits/648157ca858568b493dd4389f285228ae3673cea)